pkg/sql/logictest/testdata/logic_test/order_by

```diff
@@ -627,15 +627,13 @@ INSERT INTO bar VALUES (0, NULL), (1, NULL)
 query ITTTTT
 EXPLAIN (METADATA) SELECT * FROM bar ORDER BY baz, id
 ----
-0  scan  ·      ·          (id, baz)  +baz,key
-0  ·     table  bar@i_bar  ·          ·
-0  ·     spans  ALL        ·          ·
-
-# Here rowsort is needed because the ORDER BY clause does not guarantee any
```

Comment:
This comment was incorrect - we advertise that `ORDER BY` has "null first" semantics.